Makefile

```diff
@@ -180,13 +180,7 @@ clean:
 
 .PHONY: update-aries
 update-aries:
```

Comment:
Can this target just be removed completely?

Reply:
Not sure if it's used elsewhere, so I've left it with a deprecation notice for now.